src/features/isanteplus/patientMerge.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Patient Merge
+
+Background:
+Given User log into the system
+
+@patientMerge
+Scenario: Patient Merge
```

Comment:
@sherrif10 , does this take into account both **Patient Merge with IDs**  and **Patient Merge with name  Scenarios** ??

Reply:
Ohh yes, here is the fact, Patient Ids comes with the exact name, it would be different if the test is for IST CODE ,but for Ids and names, they take in same attribute, forexample agiven id will be by default generated by name